dm/dm.go

```diff
@@ -190,10 +192,13 @@ func execute() {
 			usage()
 		}
 
-		tUrl := getTypeUrl(args[1])
-		if tUrl == "" {
+		tUrls := getTypeUrl(args[1])
+		var tUrl = ""
+		if len(tUrls) == 0 {
 			// Type is most likely a primitive.
 			tUrl = args[1]
+		} else {
+			tUrl = tUrls[0]
```

Comment:
Are you guaranteeing that the first URL will be the one that points to the instance? How does this work with a package vs. the template type you create to encapsulate the package?